src/main/java/uk/gov/hmcts/reform/sscs/service/LetterUtils.java

```diff
@@ -197,4 +191,112 @@ public static Stream<Pair<String, YesNo>> buildOtherPartiesForReasonableAdjustme
+        if (notificationWrapper.getNotificationType().getEvent().equals(EventType.ACTION_FURTHER_EVIDENCE)) {
+            if (isValidAppellantForSetAsideRequest(notificationWrapper, subscriptionWithType)
+                    || isValidAppellantRepresentativeForSetAsideRequest(notificationWrapper, subscriptionWithType)
+                    || isValidJointPartyForSetAsideRequest(notificationWrapper, subscriptionWithType)
+                    || isValidOtherParty(notificationWrapper, subscriptionWithType)) {
+                return "confirmation";
+            } else {
+                return "notice";
+            }
+        }
+        return "";
+    }
+
+    private static boolean isValidAppellantRepresentativeForSetAsideRequest(NotificationWrapper notificationWrapper, SubscriptionWithType subscriptionWithType) {
+        return  (notificationWrapper.getNewSscsCaseData().getOriginalSender().getValue().getCode().equalsIgnoreCase("representative")
+                && (subscriptionWithType.getParty().getId().equalsIgnoreCase(notificationWrapper.getNewSscsCaseData().getAppeal().getAppellant().getId())
+                    || subscriptionWithType.getParty().getId().equalsIgnoreCase(notificationWrapper.getNewSscsCaseData().getAppeal().getRep().getId())));
+    }
+
+    private static boolean isValidAppellantForSetAsideRequest(NotificationWrapper notificationWrapper, SubscriptionWithType subscriptionWithType) {
+        return notificationWrapper.getNewSscsCaseData().getOriginalSender().getValue().getCode().equalsIgnoreCase("appellant")
+                && subscriptionWithType.getParty().getId().equalsIgnoreCase(notificationWrapper.getNewSscsCaseData().getAppeal().getAppellant().getId());
+    }
+
+    private static boolean isValidJointPartyForSetAsideRequest(NotificationWrapper notificationWrapper, SubscriptionWithType subscriptionWithType) {
+        return (notificationWrapper.getNewSscsCaseData().getOriginalSender().getValue().getCode().equalsIgnoreCase("jointParty")
+                && subscriptionWithType.getParty().getId().equalsIgnoreCase(notificationWrapper.getNewSscsCaseData().getJointParty().getId()));
+    }
+
+    static boolean isValidOtherParty(NotificationWrapper notificationWrapper, SubscriptionWithType subscriptionWithType) {
+        String subscriptionPartyId = subscriptionWithType.getPartyId();
+        String requesterId = notificationWrapper.getNewSscsCaseData().getOriginalSender().getValue().getCode();
+
+        if (nonNull(subscriptionPartyId) && requesterId.contains("otherParty")) {
+            List<CcdValue<OtherParty>> otherParties = notificationWrapper.getNewSscsCaseData().getOtherParties();
+            if (nonNull(otherParties) && isValidOtherPartyForSubscription(subscriptionPartyId, requesterId, otherParties)) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
Preempting Jarek on this. As I have commented previously, when the if condition is really wordy esp with lots of follow on chained getters, it is a good idea to extract to a more instantly readable variable boolean. 

eg
`boolean isRepresentative = notificationWrapper.getNewSscsCaseData().getOriginalSender().getValue().getCode().equalsIgnoreCase("representative")`

Reply:
Extracted to local variable wherever is possible.

Reply:
see new comment